src/com/sheepit/client/standalone/swing/activity/Settings.java

```diff
@@ -430,9 +433,16 @@ else if (config.getGPUDevice().getType().equals("OPENCL")) {
 		parent.getContentPane().add(compute_devices_panel, constraints);
 		
 		// other
-		CollapsibleJPanel advanced_panel = new CollapsibleJPanel(new GridLayout(3, 2));
+		CollapsibleJPanel advanced_panel = new CollapsibleJPanel(new GridLayout(4, 2));
 		advanced_panel.setBorder(BorderFactory.createTitledBorder("Advanced options"));
 		
+		JLabel useSysTrayLabel = new JLabel("Minimise to SysTray");
```

Comment:
```suggestion
		JLabel useSysTrayLabel = new JLabel("Minimize to SysTray");
```
Oddly enough, the commit and everything else is using `minimize` ;)

Reply:
Americans... :stuck_out_tongue: Minimise is the British way of writing :smile: and Australia is part of the Commonwealth LOL

https://preply.com/en/question/minimise-or-minimize